welcome/index.adoc

```diff
@@ -159,6 +165,8 @@ The `odo` CLI tool lets developers create single or multi-component applications
 - **xref:../cicd/pipelines/understanding-openshift-pipelines.adoc#op-key-features[Create CI/CD Pipelines]**: Pipelines are serverless, cloud-native, continuous integration and continuous deployment systems that run in isolated containers.
 They use standard Tekton custom resources to automate deployments and are designed for decentralized teams that work on microservice-based architecture.
 
+endif::openshift-rosa,openshift-dedicated,openshift-dpu[]
```

Comment:
This endif statement is new. Did you find that it is needed for all 3 distros (rosa, dedicated, dpu) for some reason? Or should it be only for dpu?